packages/react-components/react-nav-preview/src/components/NavCategoryItem/useNavCategoryItem.styles.ts

```diff
@@ -42,11 +30,5 @@ export const useNavCategoryItemStyles_unstable = (state: NavCategoryItemState):
     state.root.className,
   );
 
```

Comment:
I think you're missing the expandIcon className handle:
```jsx
state.expandIcon.className = mergeClasses(navCategoryItemClassNames.expandIcon, state.expandIcon.className);
```

Reply:
Good call - maybe double check I got what you intended?